doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,107 @@
+
+There are several methods of how to create an MVUX command.
+
+1. Using code generation
+
+    As demonstrated in previous tutorials, when creating a method in the Model, it will be generated as an `AsyncCommand` in the Bindable Model.
+    
+    Command generation rules:
+    
+     - Any parameter that has a type and name matching a Feed or a State in this Model will be evaluated when the Command is invoked and its current latest value will be passed in as an argument.  
+     You'll find this feature to be very powerful in invoking commands by combining data from various Feeds in addition to a command parameter received from the View.    
+     - Can be either synchronous or asynchronous
+     - Can have one `CancellationToken` as its last parameter, but it's not mandatory.
+
+     > [!TIP]  
+     > You can explicitly match a parameter with a Feed even the names don't match by decorating the parameter with the `FeedParameter` attribute:  
+     >
+     > ```csharp
+     > public IFeed<string> Message { get; }
+     > 
+     > public async ValueTask Share([FeedParameter(nameof(Message))] string msg)
+     > {
+     > }
+     > ```
+
+     > [!TIP]  
+     > You can also opt in or out of implicit matching of Feeds and command parameters by the decorating the current assembly or class with the `ImplicitFeedCommandParameters` attribute:  
+     >
+     > ```csharp
+     > [assembly:ImplicitFeedCommandParameter(false)]
+     >
+     > [ImplicitFeedCommandParameter(true)]
+     > public partial record MyModel
+     > ```
+
+1. Using factory methods
+
+    Commands can also built manually using the static class [`Command`](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive/Presentation/Commands/Command.cs).  
+    This class provides factory methods for creating commands.
+
+    - Async
+
+    ```csharp
+    public ICommand MyCommand => Command.Async(async(ct) => await PingServer(ct));
+    ```
+
+    > [!Note]  
+    > You need not worry about the `=>` operator and that the `Command.Async` will be called over and over, this value is cached and will only be called once.  
+    The benefit of using `=>` in place of traditional `{ get; } = ...`, is that local methods are accessible (otherwise these would have been required to be intialized in the constructor).
+
+    - `Create` & `Create<T>`;
+
+    To create a command you can use the fluent API of `ICommandBuilder` provided in the `Command.Create` factory methods, which provides the following three methods:
+
+    - Given - This method takes a Feed (or a State!) and configures a command which will be triggered whenever a new value is available to the Feed.
+
+        ```csharp
+        public IFeed<int> PageCount => ...
+
+        public IAsyncCommand MyCommand => Command.Create(builder => builder.Given(PageCount));
+        ```
+
+    - When - Limits the command execution to a set prerequisite - in other words sets the 'can execute' of the command.
```

Comment:
The answer is no, the only thing that's checked is that the parameter type matches the one in the method.
I've opened [this issue](https://github.com/unoplatform/uno.extensions/issues/1396) so that MVUX also supports implicit CanExecute hook up.

Anyway, this section talks about explicit command creation using `Command.Create`, the section above discusses generated commands and mentions how `CanExecute` only works when parameter type matches.